libs/sdk-ui-kit/README-DEV.MD

```diff
@@ -0,0 +1,275 @@
+# New Component Rules
+
+## Naming
+
+The component name should match its file name.
+
+```tsx
+./UiButton.tsx -> <UiButton />
+```
+
+## Collocation
+
+Each component should be collocated with its own styles.
+
+```tsx
+./UiButton.tsx
+./UiButton.scss
+```
+
+## Interfaces
+
+Keep interface of the components as simple as possible.
+
+Always think about how you personally would like to use the component.
+
+Interface should be aligned with Figma as much as possible - design of the interface should be discussed together with respective UX and UI personas.
+
+## Styles
+
+Styles should follow the [BEM](https://en.bem.info/methodology/quick-start/) methodology.
+
+Follow these rules:
+- Use prefixes for root elements to avoid class name collisions (e.g., `.gd-ui-kit-button`, `.gd-ui-kit-icon`).
+- Minimize nesting and overrides for better readability and maintainability
+- Avoid using margins for root elements; components should not affect the layout of their parent or sibling elements.
```

Comment:
"must not" instead of "should not" in this and the next sentence.

Reply:
Updated. I also added new readme for more details about theming (see the last commit).